README.md

```diff
@@ -41,8 +41,8 @@ it will be available again on the next boot.
 # Build and install instructions
 Standard installation using
 ```
-./bootstrap
-./configure
+bootstrap
```

Comment:
For those envs where `.` is not in $PATH these commands will fail.
I'd like to keep the `./` for bootstrap and configure.
The rest below is ok...

Reply:
Indeed, fixed.